CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+1.26.6 (2021-06-25)
+-------------------
+
+* Deprecate the urllib3.contrib.ntlmpool module. urllib3 is not able
```

Comment:
Can we use double-backticks to format the code sections:

- urllib3.contrib.ntlmpool
- Transfer-Encoding
- request_chunked()
- Retry.DEFAULT_ALLOWED_METHODS

Reply:
Thanks, fixed